pyspecProcScripts/correlation_alignment.py

```diff
@@ -33,8 +35,15 @@ def correl_align(s, align_phases=False,tol=1e-4,indirect_dim='indirect',fig_titl
                     sets the tolerance limit for the alignment procedure
     ph1_selection:  int
                     index position of the coherence pathway in phase program 1
```

Comment:
this is no longer an argument above?  it's replaced by `signal_pathway`? (which is good)